katas/adriel-gimenes/01-30/Categorize-New-Members/CategorizeNewMembers.cs

```diff
@@ -0,0 +1,13 @@
+namespace katas.AdrielGimenes;
+
+using System;
```

Comment:
Usually "using" statements are placed at the beginning of the file, then the namespace and finally the rest of the code and definitions

Reply:
Done.